### PR TITLE
100% code coverage

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: test_requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 coverage==3.7
 nose==1.3
+flake8==2.2
```
Comment (kylegibson) [inline test_requirements.txt]: Is 2.2 the latest version? How do you feel about just leaving this unpegged?

Comment (jlward) [inline test_requirements.txt]: I would much rather leave it unpegged, however @winhamwr wanted me to peg it.

Comment (kylegibson) [inline test_requirements.txt]: I'm +1 on leaving it unpegged.

**Comment by jlward** [comment]
@winhamwr says this is good to go.

File: test_requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 coverage==3.7
 nose==1.3
+flake8==2
```
Comment (kylegibson) [inline test_requirements.txt]: Why did you change this to downgrade flake8?
